app/views/coronavirus_landing_page/components/landing_page/_nhs_notice.html.erb

```diff
@@ -21,12 +21,18 @@
 
   <div class="govuk-grid-column-two-thirds">
     <% nhs_banner["sections"].each do |section| %>
-      <%= render "govuk_publishing_components/components/heading", {
-        text: section["heading"],
-        heading_level: 2,
-        font_size: "s",
-        margin_bottom: 4,
-      } if section["heading"].present? %>
+      <% if section["url"].present? %>
```

Comment:
I am wondering if we should also check the presence of heading? Incase of the unlikely event there is a url but no heading?

Reply:
Ah yes, my bad - we should absolutely check for that.  Great spot - thanks @Rosa-Fox  👏  Fixed up

Reply:
👍 thanks!